controller/api.py

```diff
@@ -5,34 +5,51 @@
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
 
 import pn532
-import prx_3hc
+import wiegand
 import store
 import payment_processor
 
 
 class Controller(object):
     def __init__(self):
-        self.prx_3hc = prx_3hc.PRX_3HC()
+        self.wiegand = wiegand.Wiegand26()
         self.pn532 = pn532.PN532()
         self.store = store.Store()
         self.payment_processor = payment_processor.Payment_Processor()
 
     def setup(self):
         print("controller is setting up")
 
-        self.prx_3hc.setup()
+        self.wiegand.setup()
         self.pn532.setup()
         self.store.setup()
         self.payment_processor.setup()
 
+    
+    def get_card(self):
+        """function handles exceptions. this will probably be deleted later
+           but it illustrates how the errors work
+        """
+        try:
+            return self.wiegand.read()
+        except FacilityError:
+            print("Please use a Bennington Card")
+            return self.get_card()
+        except ReadError:
+            print("Read Error: Please try again")
+            return self.get_card()
+
     def run(self):
         """run the system sequentially (only for demo purposes)
            return False whenever an error is occured to end the session
         """
         print("controller is running")
 
         # tap a bennington card to start a check out session
-        bennington_card = self.prx_3hc.read()
+
+        bennington_card = self.get_card()
```

Comment:
as mentioned above, the error handling will be put in later. i'm fine if it's just `self.wiegand.read()` for now.